💄카테고리 페이지 UI 구현

Puterism opened the pull request:
![image](https://user-images.githubusercontent.com/2542730/98773420-3ddd3400-242c-11eb-9fa5-ac693a59c094.png)
와이어프레임 기반으로 이런 화면을 만들었습니다

- 스포카 한 산스를 Fallback 폰트로 설정
- Sidebar의 Link를 react-router-dom의 Link로 변경
- 반응형으로 구현하지는 않았습니다. 추후 반응형으로 바꾸거나 창의 크기를 줄여도 UI가 깨지지 않도록 수정할 예정입니다.

추가로 `Layout` 컴포넌트의 Grid가 5칸으로 너무 적은 것 같아서 `Layout`을 사용하지 않고 Grid를 8칸으로 나눴습니다.
`Layout` 컴포넌트의 Grid도 8칸으로 나누는 건 어떨까요?

Puterism committed:
💄카테고리 페이지 UI 구현
- 스포카 한 산스를 Fallback 폰트로 설정
- Sidebar의 Link를 react-router-dom의 Link로 변경

simsimjae commented on src/components/Card/index.js:
```diff
@@ -0,0 +1,114 @@
+            <Box backgroundColor="#2699fb" opacity="0.6" position="absolute" top="0" left="0" width="100%" height="100%" zIndex="1" />
+            <Box
+              zIndex="10"
+              margin="63px 69px"
```
차크라에서 기본으로 제공하는 4단위 그리드가 있어서 px을 없애고 숫자만 넣으면 테마에서 관리할수있긴한데
px로 하는게 편해서 이렇게 하신건가여?

https://next.chakra-ui.com/docs/theming/theme

simsimjae replied:
기본으로 제공하는 테마에 오버라이드하는방식으로 이런 값들을 제어하면 좋을거같아요 차크라가 그런 의도로 설계된 라이브러리여서요~

Puterism replied:
https://next.chakra-ui.com/docs/theming/theme#spacing
말씀하신 부분이 이 spacing인가요?

Puterism replied:
제플린에 맞추어서 구현하다 보니 px 단위로 적게 됐습니다
차크라가 단순히 컴포넌트에 prop으로 바로 스타일을 적용하기 위한 것인 줄 알았는데 그게 아니었군요...

말씀하신대로 테마에서 값을 관리하도록 코드 수정해서 커밋 올려보겠습니다
사용 예시가 맞는지 한번 확인 부탁드릴게요

simsimjae reviewed (commented):
수고하셨습니다. 네 레이아웃 그리드 8칸으로 하는거 좋습니다

Puterism commented:
> 수고하셨습니다. 네 레이아웃 그리드 8칸으로 하는거 좋습니다

Layout 컴포넌트 수정해서 다시 커밋 푸시하겠습니다

Puterism committed:
💄Card UI 구현 마무리
- Layout Grid 8칸으로 수정
- index.html에서 lang을 ko로 수정
- Chakra theme에 맞게 margin, padding 등 수정

Puterism committed:
🚚conflict 없는 master 머지를 위한 pages 경로 수정

simsimjae commented on src/components/Card/index.js:
```diff
@@ -13,11 +13,32 @@ const Card = (props) => {
+            color="#ffffff"
+          >
+            <Box
+              backgroundColor="#2699fb"
```
컬러도 테마로 관리하는게 좋긴합니다~ 이것만 수정해서 머지해주세요

Puterism replied:
https://github.com/depromeet/8th-final-front-6team/pull/2/commits/73e0367ee4017c40306d0efbca93a8d829e31da0 에서 수정 완료했습니다

Puterism committed:
🎨 Chakra UI theme에서 color가 관리되도록 수정

Puterism committed:
Merge branch 'master' into feature/category
